src/In.ProjectEKA.HipService/Gateway/GatewayPathConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace In.ProjectEKA.HipService.Gateway
+{
+    public class GatewayPathConstants
+    {
+        public static string OnDiscoverPath = "/care-contexts/on-discover";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'OnDiscoverPath' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5570468)